ui/app/helpers/utils/util.js

```diff
@@ -407,3 +407,43 @@ export function constructTxParams ({ sendToken, data, to, amount, from, gas, gas
   }
   return addHexPrefixToObjectValues(txParams)
 }
+
+/**
+ * Makes a JSON RPC request to the given URL, with the given RPC method and params.
+ *
+ * @param {string} rpcUrl - The RPC endpoint URL to target.
+ * @param {string} rpcMethod - The RPC method to request.
+ * @param {Array<unknown>} [rpcParams] - The RPC method params.
+ * @returns {Promise<unknown|undefined>} Returns the result of the RPC method call,
```

Comment:
Nit: `unknown|undefined` seems redundant :thinking:  since `undefined` is a valid `unknown` value

```suggestion
 * @returns {Promise<unknown>} Returns the result of the RPC method call,
```